### PR TITLE
error codes

### PR DESCRIPTION
**Pull request by thesimplekid**
Tracking progress:
- [ ] nutshell
- [x] CDK
- [ ] cashu-ts
- [ ] nutmix
- [ ] gonuts
- [ ] ...

### CODE REVIEW

File: 03.md [hunk 1]
```diff
@@ -78,6 +78,13 @@ If successful, `Bob` will respond with a `PostSwapResponse`
 }
 ```
 
+## Errors
+
+| Code   | Description                               |
+|--------|-------------------------------------------|
+| 11001  | Token is already spent                    |
+| 10002  | Blinded message of output already signed  |
+
```
Comment (elnosh) [inline 03.md:87]: I have an error if the mint could not verify a proof provided. Let me know if that one would be good to add here.

File: 02.md [hunk 2]
```diff
@@ -191,6 +191,14 @@ A useful flow is:
...
 - If any of the keysets has changed its `active` flag, update it in the db and use the keyset accordingly
 
+## Errors
+
+| Code   | Description                        |
+|--------|------------------------------------|
+| 11002  | Insufficant fee included           |
+| 11003  | Fee over paid                      |
+| 12001  | Keyset is not known                |
+
```
Comment (elnosh) [inline 02.md:201]: Here I think there could be error if mint receives request to sign outputs from an inactive keyset.

File: 02.md [hunk 3]
```diff
@@ -191,6 +191,14 @@ A useful flow is:
 - For all new keyset returned here which we don't have yet, get it using `GET /v1/keys/{keyset_id}` and store it
 - If any of the keysets has changed its `active` flag, update it in the db and use the keyset accordingly
 
+## Errors
+
+| Code   | Description                        |
+|--------|------------------------------------|
+| 11002  | Insufficant fee included           |
```
Comment (elnosh) [inline 02.md:198]:
```suggestion
| 11002  | Insufficient fee included           |
```

**Comment by thesimplekid** [comment]
Error codes not defined in nutshell.
```
10002 => Self::BlindedMessageAlreadySigned,
10003 => Self::TokenNotVerified,
11002 => Self::InsufficientFee,
12002 => Self::KeysetInactive,
20002 => Self::TokensAlreadyIssued,
20003 => Self::MintingDisabled,
20005 => Self::QuotePending,
20006 => Self::InvoiceAlreadyPaid.
```

File: 04.md [hunk 4]
```diff
@@ -212,6 +212,18 @@ Example `MintMethodSetting`:
...
+| Code   | Description                               |
+|--------|-------------------------------------------|
+| 10002  | Blinded message of output already signed  |
+| 11005  | Unit in request is not supported          |
+| 12002  | Keyset is inactive, cannot sign messages  |
+| 20001  | Quote request is not paid                 |
+| 20002  | Tokens have already been issued for quote |
+| 20003  | Minting is disabled                       |
+
+
```
Comment (thesimplekid) [inline 04.md:226]: Add request over/under limit

File: 05.md [hunk 5]
```diff
@@ -191,6 +191,18 @@ Example `MeltMethodSetting`:
...
+
+| Code   | Description                               |
+|--------|-------------------------------------------|
+| 10002  | Blinded message of output already signed  |
+| 10003  | Token could not be verified               |
+| 11001  | Token is already spent                    |
+| 11005  | Unit in request is not supported          |
+| 20005  | Quote is pending                          |
+| 20006  | Invoice already paid                      |
+
```
Comment (thesimplekid) [inline 05.md:204]: Add request over/under limit

**Comment by thesimplekid** [comment]
close for #161

**Pull request closed without merging**